.github/workflows/build-images.yml

```
@@ -296,8 +302,14 @@ jobs:
         run: |
           rm -rf "scripts/ci"
           mv "main-airflow/scripts/ci" "scripts"
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.7'
+          cache: 'pip'
+      - run: pip3 install --editable ./dev/breeze
```

```suggestion
      - run: python -m pip install --editable ./dev/breeze
```

We should leave PIP here. From what I saw recently in discussions - PIP plans to deprecate old usage and they prefer to use `python -m pip` instead: https://github.com/pypa/pip/issues/3164

---

And everywhere else too.

---

I see, should we assume that we have just a python version installed? Or we should specify the version:
` python -m pip install --editable ./dev/breeze`

---

Python version is installed in the previous step (3.7) so python is good (and it will be pythoni 3.7.